src/coreclr/debug/di/publish.cpp

```diff
@@ -81,7 +81,7 @@ BOOL GetAllProcessesInSystem(DWORD *ProcessId,
 
     } while ((succ == TRUE) && (iIndex < (int)dwArraySize));
 
-    // I would like to know if we're running more than 512 processes on Win95!!
+    // I would like to know if we're running more than 512 processes!!
```

Comment:
Who is "I" and why do they want to know that? Strange comment :-)

Reply:
gone